test/transcoding/OpPhi_ArgumentsPlaceholders.ll

```diff
@@ -23,16 +23,16 @@
 target datalayout = "e-p:32:32-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024"
 target triple = "spir"
 
-%struct.Node = type { %struct.Node.0 addrspace(1)* }
+%struct.Node = type { ptr addrspace(1) }
```

Comment:
Is `%struct.Node.0` really unneeded here? then looks like the line below can be removed, if I understand it right

Reply:
Good catch, I've removed it.